src/components/PanelV2/__tests__/PanelV2.spec.js

```diff
@@ -3,38 +3,121 @@
  * @copyright IBM Security 2019
  */
 
-import { mount } from 'enzyme';
-import React, { Fragment } from 'react';
-
-import { label, onClick } from '../_mocks_';
+import { cleanup, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
 
 import { Button, PanelV2, PanelContent } from '../../..';
 
 describe('PanelV2', () => {
-  it('renders', () => {
-    const wrapper = mount(
+  afterEach(cleanup);
+
+  test('should render as expected', async () => {
+    render(
       <PanelV2
-        title={label}
-        subtitle={label}
-        closeButton={{ onClick }}
-        renderFooter={() => (
-          <Fragment>
-            <Button
-              id="example-secondary-button"
-              kind="secondary"
-              onClick={onClick}
-            >
-              Close
-            </Button>
-            <Button id="example-primary-button" onClick={onClick}>
-              Add
-            </Button>
-          </Fragment>
-        )}
+        title="test title"
+        subtitle="test subtitle"
+        closeButton={{
+          label: 'test close',
+        }}
+        renderFooter={() => <Button>test footer button</Button>}
       >
-        <PanelContent>{label}</PanelContent>
+        <PanelContent>test content</PanelContent>
       </PanelV2>
     );
-    expect(wrapper).toMatchSnapshot();
+    await expect(document.body).toMatchSnapshot();
+  });
+
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
+    render(
+      <PanelV2
+        title="test title"
+        subtitle="test subtitle"
+        closeButton={{
+          label: 'test close',
+        }}
+        renderFooter={() => <Button>test footer button</Button>}
+      >
+        <PanelContent>test content</PanelContent>
+      </PanelV2>,
+      {
+        // DAP requires a landmark '<main>' in the DOM:
+        container: document.body.appendChild(main),
+      }
+    );
+    await expect(document.body).toHaveNoAxeViolations();
+    await expect(document.body).toHaveNoDAPViolations('PanelV2');
+  });
+
+  test('should invoke close mock when close button is clicked', () => {
+    const onCloseMock = jest.fn();
+    const { getByLabelText } = render(
+      <PanelV2
+        closeButton={{
+          label: 'test close',
+          onClick: onCloseMock,
+        }}
+      />
+    );
+
+    userEvent.click(getByLabelText(/test close/i));
+    expect(onCloseMock).toHaveBeenCalled();
```

Comment:
Sometimes I take the assertion a bit further by using `toHaveBeenCalledTimes(1)` just in case so it doesn't call it multiple times.